package.json

```diff
@@ -4,17 +4,17 @@
   "description": "",
   "main": "./build/app.js",
   "scripts": {
-    "build": "rm -rf build && parallelshell 'npm run build:js' 'npm run build:style'",
+    "build": "rimraf build && parallelshell 'npm run build:js' 'npm run build:style'",
     "build:js": "babel src --out-dir build",
     "build:style": "node-sass --output-style compressed --source-map build/james.css.map style/main.scss build/james.css && autoprefixer build/james.css && npm run copy-files",
     "watch": "parallelshell 'npm run watch:js' 'npm run watch:style'",
     "watch:js": "nodemon --watch src --ext js --no-stdin --exec \"npm run build:js\"",
     "watch:style": "nodemon --watch style --ext scss --no-stdin --exec \"npm run build:style\"",
-    "copy-files": "rm -rf build/fonts && mkdir -p build/fonts && cp ./node_modules/font-awesome/fonts/* build/fonts/ && cp -r ./resource/* ./build/",
-    "package": "npm run build && rm -rf ./package &&  mkdir -p ./package && mkdir -p ./binaries && browserify --node -e build/index.js -o ./package/index.js --exclude remote && npm run -s copy-files-package && npm run -s create-binaries",
+    "copy-files": "rimraf build/fonts && mkdirp build/fonts && cp ./node_modules/font-awesome/fonts/* build/fonts/ && cp -r ./resource/* ./build/",
+    "package": "npm run build && rimraf ./package &&  mkdirp ./package && mkdirp ./binaries && browserify --node -e build/index.js -o ./package/index.js --exclude remote && npm run -s copy-files-package && npm run -s create-binaries",
     "create-binaries": "electron-packager ./package James --all --overwrite --icon=./resource/icon.icns --version=0.28.3 --out binaries",
     "zip-binaries": "cd binaries; rm *.zip; for i in *; do zip --symlinks -r \"${i%/}.zip\" \"$i\"; done",
```

Comment:
It's not only the zip command but the whole scripting in there needs to be rewritten with node